### PR TITLE
README tweaks per @ajvb

### PR DESCRIPTION
**Pull request by novwhisky**

### CODE REVIEW

**Comment by ajvb** [comment]
Hey @wittnl, thanks for the PR. I believe that either `npm run start` or how the go binary works in development is messed up. `npm run build` is for building production builds, and does not do automatic reloading while developing.

**Comment by novwhisky** [comment]
I think the issue is `npm start` invokes [webpack-dev-server](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/scripts/start.js#L75) with a virtual fs and `./pursuance` is expecting the presence of `build/index.html` which won't be written to. Perhaps the go binary could fall back to looking at localhost:8080?

File: README.md [hunk 1]
```diff
@@ -55,7 +55,9 @@ Linux or OS X.)
 Then, in another terminal session run this (**an error about not
 finding `github.com/PursuanceProject/pursuance` is OK here**) --
 
-``` $ go get ./... ```
+``` $ go get ./.. ```
```
Comment (elimisteve) [inline README.md]: @wittnl This still needs the full 3 `.`s, but good point about `npm run build` :+1:

File: README.md [hunk 2]
```diff
@@ -55,7 +55,9 @@ Linux or OS X.)
...
+
+``` $ npm run build ```
```
Comment (elimisteve) [inline README.md]: :+1:

File: README.md (hunk 1 above)
Comment (novwhisky) [reply README.md]: Ah it's a go thing, I'll revert that.

**Comment by elimisteve** [comment]
@wittnl Merged :tada: . Thanks!

**Pull request merged**